src/internal/m365/collection/drive/collections_tree.go

```diff
@@ -227,7 +247,321 @@ func (c *Collections) makeDriveCollections(
 	// 	}
 	// }
 
-	return nil, nil, pagers.DeltaUpdate{}, clues.New("not yet implemented")
+	return nil, nil, du, errTreeNotImplemented
+}
+
+// populateTree constructs a new tree and populates it with items
+// retrieved by enumerating the delta query for the drive.
+func (c *Collections) populateTree(
+	ctx context.Context,
+	tree *folderyMcFolderFace,
+	limiter *pagerLimiter,
+	stats *driveEnumerationStats,
+	drv models.Driveable,
+	prevPaths map[string]string,
+	prevDeltaLink string,
+	counter *count.Bus,
+	errs *fault.Bus,
+) (pagers.DeltaUpdate, error) {
+	ctx = clues.Add(ctx, "invalid_prev_delta", len(prevDeltaLink) == 0)
+
+	logger.Ctx(ctx).Infow(
+		"running backup",
+		"limits", c.ctrl.PreviewLimits,
+		"effective_limits", limiter.effectiveLimits(),
+		"preview_mode", limiter.enabled())
+
+	var (
+		driveID = ptr.Val(drv.GetId())
+		el      = errs.Local()
+	)
+
+	// TODO(keepers): to end in a correct state, we'll eventually need to run this
+	// query multiple times over, until it ends in an empty change set.
```

Comment:
hum, that's a good question.  Can I hand wave and say, "I'll definitely figure something out because I'm going to have to have something figured out"?